channels/client/addin.c

```diff
@@ -212,7 +212,13 @@ static FREERDP_ADDIN** freerdp_channels_list_dynamic_addins(LPCSTR pszName, LPCS
 	NativePathCchAppendA(pszSearchPath, cchSearchPath + 1, pszAddinPath);
 	NativePathCchAppendA(pszSearchPath, cchSearchPath + 1, pszPattern);
 	free(pszPattern);
-	hFind = FindFirstFileA(pszSearchPath, &FindData);
+	{
+		WCHAR* wpath = NULL;
```

Comment:
Moving this up (where `FindData` already is defined) and avoiding the separate scope would make the code better readable imo

Reply:
ok, moved to a separate function.